schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+    id INT,
```

Comment:
- [OPTIONAL] I think it would be better if you make the id column not null and autoincremented like  `id int NOT NULL IDENTITY(1,1)` and also if you mark the id column as a primary key like `PRIMARY KEY (id)`